spec/services/datetime_parser_spec.rb

```diff
@@ -70,22 +72,21 @@
       parser = DatetimeParser.new(params)
       parser.parse
 
-      expect(parser.issues.items).to be_empty
+      expect(parser.issues).to be_empty
     end
 
     it "returns issues when the date/time are blank" do
       parser = DatetimeParser.new(date: nil, time: nil, issue_prefix: :schedule)
       parser.parse
 
-      date_form_message = parser.issues.items_for(:schedule_date).first[:text]
-      expect(date_form_message).to eq(
-        I18n.t!("requirements.schedule_date.invalid.form_message"),
-      )
+      relevant_issues = parser.issues_for(:schedule_date)
+      expect(relevant_issues.count).to eq(1)
+
+      date_form_message = relevant_issues.first
+      expect(date_form_message).to eq("Enter a valid date")
```

Comment:
We don't have any tests that this error message actually appears in the page. (Nor its placement in the page, i.e. inline in the relevant component, or in the 'global' error at the top of the page).

Wondering if it's worth doing or if these unit tests are sufficient.